Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -52,9 +52,40 @@ DECLARE_SOA_COLUMN(NumContrib, numContrib, uint32_t);
 DECLARE_SOA_COLUMN(CollisionTime, collisionTime, float);
 DECLARE_SOA_COLUMN(CollisionTimeRes, collisionTimeRes, float);
 DECLARE_SOA_COLUMN(CollisionTimeMask, collisionTimeMask, uint8_t); // TODO put nature of CollisionTimeRes here, e.g. MSB 0 = exact range / 1 = Gaussian uncertainty
+DECLARE_SOA_COLUMN(CollisionTime0, collisionTime0, float);
+DECLARE_SOA_COLUMN(CollisionTimeRes0, collisionTimeRes0, float);
+DECLARE_SOA_COLUMN(CollisionTime1, collisionTime1, float);
+DECLARE_SOA_COLUMN(CollisionTimeRes1, collisionTimeRes1, float);
+DECLARE_SOA_COLUMN(CollisionTime2, collisionTime2, float);
+DECLARE_SOA_COLUMN(CollisionTimeRes2, collisionTimeRes2, float);
+DECLARE_SOA_COLUMN(CollisionTime3, collisionTime3, float);
+DECLARE_SOA_COLUMN(CollisionTimeRes3, collisionTimeRes3, float);
+DECLARE_SOA_COLUMN(CollisionTime4, collisionTime4, float);
+DECLARE_SOA_COLUMN(CollisionTimeRes4, collisionTimeRes4, float);
+DECLARE_SOA_COLUMN(CollisionTime5, collisionTime5, float);
+DECLARE_SOA_COLUMN(CollisionTimeRes5, collisionTimeRes5, float);
+DECLARE_SOA_COLUMN(CollisionTime6, collisionTime6, float);
+DECLARE_SOA_COLUMN(CollisionTimeRes6, collisionTimeRes6, float);
+DECLARE_SOA_COLUMN(CollisionTime7, collisionTime7, float);
+DECLARE_SOA_COLUMN(CollisionTimeRes7, collisionTimeRes7, float);
+DECLARE_SOA_COLUMN(CollisionTime8, collisionTime8, float);
+DECLARE_SOA_COLUMN(CollisionTimeRes8, collisionTimeRes8, float);
+DECLARE_SOA_COLUMN(CollisionTime9, collisionTime9, float);
+DECLARE_SOA_COLUMN(CollisionTimeRes9, collisionTimeRes9, float);
```

Comment:
This you can of course change to be an array column. Any reason you did not do so?

Reply:
Yes, you are right, I wanted to have the PR open with the code that I had locally.
I do not intend to have those added to the AF, but I agree that if I wanted to add them I would use an array, as you suggested yesterday.

Reply:
Ok. Shall we do that to unblock you at least on that part? We can keep the other as is to track the issue. I did some investigations yesterday and indeed it seems to be some issue in the implementation of `std::tuple<>` in GCC. If you accept the PR I did to your branch (or you allow me to edit this) we might be able to get more insights.

Reply:
Hi @ktf, I should have merged the PR you opened on my branch, however you started from the master branch while I included your change into the dev version.
In principle the PR is already open to edits by reviewers.